src/algebra/monoid_algebra/basic.lean

```diff
@@ -684,6 +694,14 @@ lemma lift_unique (F : monoid_algebra k G →ₐ[k] A) (f : monoid_algebra k G)
   F f = f.sum (λ a b, b • F (single a 1)) :=
 by conv_lhs { rw lift_unique' F, simp [lift_apply] }
 
+/--  A multiplicative homomorphism `f : G →* H` between two monoids induces a `k`-algebra
+homomorphism `map_domain_alg_hom k A f : monoid_algebra k G →ₐ[k] monoid_algebra k H`. -/
+def map_domain_alg_hom (k A : Type*) [comm_semiring k] [semiring A] [algebra k A] {H : Type*}
+  [monoid H] (f : G →* H) :
```

Comment:
This one can generalize to `monoid_hom_class`

Reply:
I think that these are all changed to `appropriate_hom_class`es.

Reply:
This one still says `G →* H`